2021-01-16 16:05:04,783 INFO    MainThread:2196144 [internal.py:wandb_internal():64] W&B internal server running at pid: 2196144
2021-01-16 16:05:04,785 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: check_version
2021-01-16 16:05:04,785 INFO    WriterThread:2196144 [datastore.py:open_for_write():77] open: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/run-dn161w34.wandb
2021-01-16 16:05:04,786 DEBUG   SenderThread:2196144 [sender.py:send():117] send: header
2021-01-16 16:05:04,786 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:05:04,786 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: check_version
2021-01-16 16:05:04,791 DEBUG   Thread-9  :2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): pypi.org:443
2021-01-16 16:05:05,483 DEBUG   Thread-9  :2196144 [connectionpool.py:_make_request():442] https://pypi.org:443 "GET /pypi/wandb/json HTTP/1.1" 200 52801
2021-01-16 16:05:05,705 DEBUG   SenderThread:2196144 [sender.py:send():117] send: run
2021-01-16 16:05:05,714 DEBUG   SenderThread:2196144 [git.py:repo():31] git repository is invalid
2021-01-16 16:05:05,726 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:05:06,105 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:05:06,111 INFO    SenderThread:2196144 [dir_watcher.py:__init__():165] watching files in: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files
2021-01-16 16:05:06,112 INFO    SenderThread:2196144 [sender.py:_start_run_threads():596] run started: dn161w34 with start time 1610784303
2021-01-16 16:05:06,117 DEBUG   SenderThread:2196144 [sender.py:send():117] send: summary
2021-01-16 16:05:06,117 INFO    SenderThread:2196144 [sender.py:_save_file():682] saving file wandb-summary.json with policy end
2021-01-16 16:05:06,119 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: run_start
2021-01-16 16:05:06,121 DEBUG   HandlerThread:2196144 [meta.py:__init__():34] meta init
2021-01-16 16:05:06,121 DEBUG   HandlerThread:2196144 [meta.py:__init__():48] meta init done
2021-01-16 16:05:06,121 DEBUG   HandlerThread:2196144 [meta.py:probe():190] probe
2021-01-16 16:05:06,127 DEBUG   HandlerThread:2196144 [git.py:repo():31] git repository is invalid
2021-01-16 16:05:06,128 DEBUG   HandlerThread:2196144 [meta.py:_save_pip():52] save pip
2021-01-16 16:05:06,128 DEBUG   HandlerThread:2196144 [meta.py:_save_pip():66] save pip done
2021-01-16 16:05:06,128 DEBUG   HandlerThread:2196144 [meta.py:probe():231] probe done
2021-01-16 16:05:06,134 DEBUG   SenderThread:2196144 [sender.py:send():117] send: files
2021-01-16 16:05:06,135 INFO    SenderThread:2196144 [sender.py:_save_file():682] saving file wandb-metadata.json with policy now
2021-01-16 16:05:06,142 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: status
2021-01-16 16:05:06,144 DEBUG   Thread-16 :2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:05:06,145 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:05:06,145 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: status
2021-01-16 16:05:06,154 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:05:06,513 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:05:06,518 DEBUG   Thread-16 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:05:06,524 DEBUG   Thread-16 :2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): storage.googleapis.com:443
2021-01-16 16:05:07,073 DEBUG   Thread-16 :2196144 [connectionpool.py:_make_request():442] https://storage.googleapis.com:443 "PUT /wandb-production.appspot.com/whiteashes/fedml/dn161w34/wandb-metadata.json?Expires=1610870706&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=N85INh1HaR1YS8sVzbQOKFV6wuWgs4Zm22yniihgPQPlGctTTNEchUDpMCZrfc4D5klgXOxgzKb8HxKuI4yy1KcW604WrIRSetEhts9okjPl3rugGDK6Ml1RIsmiA9qycQoExYeb6jyY6eDZ4zDJ1C150qmDkLx3ZVrcZ%2FLpc1zaoWztnYMyg2GPi7GhKKpspc12DHpfH9GZQepAdX3kIY68CWhaXDB%2FDbHafnK99w87OD5z4bEHYEBWHAJJAPTrLtD0Xf%2FAHXHWn2Ut96KqFmVpD%2BGhSbG3X8QT8PSn5eWkUwQ5jI%2B29e0QHa7A5rhPevF30MviCCpU1GkPJIRmXg%3D%3D HTTP/1.1" 200 0
2021-01-16 16:05:07,074 INFO    Thread-16 :2196144 [upload_job.py:push():131] Uploaded file /tmp/tmpz15x9ux7wandb/38b0vsw9-wandb-metadata.json
2021-01-16 16:05:07,116 INFO    Thread-13 :2196144 [dir_watcher.py:_on_file_created():213] file/dir created: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/output.log
2021-01-16 16:05:07,116 INFO    Thread-13 :2196144 [dir_watcher.py:_on_file_created():213] file/dir created: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/wandb-metadata.json
2021-01-16 16:05:07,116 INFO    Thread-13 :2196144 [dir_watcher.py:_on_file_created():213] file/dir created: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/requirements.txt
2021-01-16 16:05:07,117 INFO    Thread-13 :2196144 [dir_watcher.py:_on_file_created():213] file/dir created: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/wandb-summary.json
2021-01-16 16:05:08,520 DEBUG   Thread-10 :2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:05:08,969 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:05:09,507 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:05:11,792 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:05:13,963 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:05:14,235 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:05:18,416 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:05:18,683 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:05:22,868 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:05:23,148 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:05:27,318 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:05:27,608 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:05:31,769 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:05:32,042 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:05:32,276 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: status
2021-01-16 16:05:32,276 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:05:32,276 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: status
2021-01-16 16:05:32,281 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:05:32,696 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:05:36,222 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:05:36,483 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:05:40,674 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:05:40,934 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:05:45,124 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:05:45,383 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:05:47,698 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: status
2021-01-16 16:05:47,698 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:05:47,698 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: status
2021-01-16 16:05:47,703 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:05:48,117 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:05:49,577 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:05:49,842 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:05:54,027 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:05:54,287 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:05:58,488 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:05:58,753 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:05:59,134 INFO    Thread-13 :2196144 [dir_watcher.py:_on_file_modified():226] file/dir modified: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/output.log
2021-01-16 16:06:01,014 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:06:02,135 INFO    Thread-13 :2196144 [dir_watcher.py:_on_file_modified():226] file/dir modified: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/output.log
2021-01-16 16:06:02,946 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:06:03,120 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: status
2021-01-16 16:06:03,120 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:06:03,120 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: status
2021-01-16 16:06:03,126 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:06:03,237 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:06:03,544 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:06:07,405 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:06:11,856 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:06:16,307 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:06:16,570 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:06:18,547 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: status
2021-01-16 16:06:18,547 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:06:18,547 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: status
2021-01-16 16:06:18,552 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:06:18,972 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:06:20,762 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:06:25,144 INFO    Thread-13 :2196144 [dir_watcher.py:_on_file_modified():226] file/dir modified: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/output.log
2021-01-16 16:06:25,227 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:06:29,038 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:06:29,146 INFO    Thread-13 :2196144 [dir_watcher.py:_on_file_modified():226] file/dir modified: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/output.log
2021-01-16 16:06:29,686 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:06:33,976 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: status
2021-01-16 16:06:33,976 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:06:33,976 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: status
2021-01-16 16:06:33,983 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:06:34,400 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:06:34,401 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:06:36,149 INFO    Thread-13 :2196144 [dir_watcher.py:_on_file_modified():226] file/dir modified: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/output.log
2021-01-16 16:06:37,149 INFO    Thread-13 :2196144 [dir_watcher.py:_on_file_modified():226] file/dir modified: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/output.log
2021-01-16 16:06:38,602 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:06:43,061 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:06:43,328 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:06:47,513 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:06:49,403 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: status
2021-01-16 16:06:49,403 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:06:49,403 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: status
2021-01-16 16:06:49,410 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:06:50,160 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:06:51,971 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:06:56,434 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:06:56,799 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:07:00,890 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:07:05,163 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: status
2021-01-16 16:07:05,163 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:07:05,163 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: status
2021-01-16 16:07:05,168 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:07:06,035 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:07:06,036 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:07:09,794 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:07:10,158 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:07:14,246 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:07:18,699 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:07:21,037 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: status
2021-01-16 16:07:21,038 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:07:21,045 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: status
2021-01-16 16:07:21,050 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:07:33,803 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:07:35,750 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:07:51,407 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:08:01,608 ERROR   SenderThread:2196144 [retry.py:__call__():111] Retry attempt failed:
Traceback (most recent call last):
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/urllib3/connection.py", line 160, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
socket.timeout: timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/urllib3/connectionpool.py", line 677, in urlopen
    chunked=chunked,
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/urllib3/connectionpool.py", line 381, in _make_request
    self._validate_conn(conn)
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/urllib3/connectionpool.py", line 976, in _validate_conn
    conn.connect()
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/urllib3/connection.py", line 308, in connect
    conn = self._new_conn()
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/urllib3/connection.py", line 167, in _new_conn
    % (self.host, self.timeout),
urllib3.exceptions.ConnectTimeoutError: (<urllib3.connection.HTTPSConnection object at 0x7f7feaf16310>, 'Connection to api.wandb.ai timed out. (connect timeout=10)')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/requests/adapters.py", line 449, in send
    timeout=timeout
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/urllib3/connectionpool.py", line 725, in urlopen
    method, url, error=e, _pool=self, _stacktrace=sys.exc_info()[2]
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/urllib3/util/retry.py", line 439, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f7feaf16310>, 'Connection to api.wandb.ai timed out. (connect timeout=10)'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/wandb/old/retry.py", line 96, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/wandb/sdk/internal/internal_api.py", line 131, in execute
    return self.client.execute(*args, **kwargs)
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.py", line 38, in execute
    request = requests.post(self.url, **post_args)
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/requests/api.py", line 119, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/requests/sessions.py", line 530, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/requests/sessions.py", line 643, in send
    r = adapter.send(request, **kwargs)
  File "/home/whiteashes/anaconda3/envs/fedml/lib/python3.7/site-packages/requests/adapters.py", line 504, in send
    raise ConnectTimeout(e, request=request)
requests.exceptions.ConnectTimeout: HTTPSConnectionPool(host='api.wandb.ai', port=443): Max retries exceeded with url: /graphql (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f7feaf16310>, 'Connection to api.wandb.ai timed out. (connect timeout=10)'))
2021-01-16 16:08:06,366 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:08:14,099 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:08:14,101 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:14,102 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:14,102 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:14,102 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:14,102 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:14,103 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:14,103 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:14,103 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:14,104 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:14,104 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:14,104 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:14,105 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:16,578 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:17,827 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:08:18,245 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:08:21,029 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:25,481 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:29,106 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: status
2021-01-16 16:08:29,106 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:08:29,106 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: status
2021-01-16 16:08:29,111 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:08:29,548 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:08:29,931 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:30,197 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:08:34,383 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:38,835 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:43,288 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:43,543 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:08:44,550 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: status
2021-01-16 16:08:44,550 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:08:44,550 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: status
2021-01-16 16:08:44,555 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:08:45,000 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:08:47,740 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:52,194 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:56,650 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:08:56,911 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:09:00,002 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: status
2021-01-16 16:09:00,003 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:09:00,003 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: status
2021-01-16 16:09:00,008 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:09:00,437 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:09:01,103 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:09:05,556 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:09:10,007 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:09:10,275 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:09:12,065 WARNING MainThread:2196144 [internal.py:wandb_internal():118] Internal process interrupt: 1
2021-01-16 16:09:12,067 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: poll_exit
2021-01-16 16:09:12,067 DEBUG   SenderThread:2196144 [sender.py:send():117] send: telemetry
2021-01-16 16:09:12,085 DEBUG   SenderThread:2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:09:12,547 DEBUG   SenderThread:2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:09:12,557 DEBUG   SenderThread:2196144 [sender.py:send():117] send: exit
2021-01-16 16:09:12,557 INFO    SenderThread:2196144 [sender.py:send_exit():195] handling exit code: 0
2021-01-16 16:09:12,557 INFO    SenderThread:2196144 [sender.py:send_exit():203] send defer
2021-01-16 16:09:12,557 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:09:12,557 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: poll_exit
2021-01-16 16:09:12,558 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: defer
2021-01-16 16:09:12,558 INFO    HandlerThread:2196144 [handler.py:handle_request_defer():69] handle defer: 0
2021-01-16 16:09:12,558 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:09:12,558 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: defer
2021-01-16 16:09:12,558 INFO    SenderThread:2196144 [sender.py:send_request_defer():212] handle sender defer: 0
2021-01-16 16:09:12,558 INFO    SenderThread:2196144 [sender.py:send_request_defer():248] send defer: 1
2021-01-16 16:09:12,559 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: defer
2021-01-16 16:09:12,559 INFO    HandlerThread:2196144 [handler.py:handle_request_defer():69] handle defer: 1
2021-01-16 16:09:12,641 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:09:12,642 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: defer
2021-01-16 16:09:12,642 INFO    SenderThread:2196144 [sender.py:send_request_defer():212] handle sender defer: 1
2021-01-16 16:09:12,642 INFO    SenderThread:2196144 [sender.py:send_request_defer():248] send defer: 2
2021-01-16 16:09:12,642 DEBUG   SenderThread:2196144 [sender.py:send():117] send: stats
2021-01-16 16:09:12,642 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: defer
2021-01-16 16:09:12,642 INFO    HandlerThread:2196144 [handler.py:handle_request_defer():69] handle defer: 2
2021-01-16 16:09:12,643 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:09:12,643 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: defer
2021-01-16 16:09:12,643 INFO    SenderThread:2196144 [sender.py:send_request_defer():212] handle sender defer: 2
2021-01-16 16:09:12,643 INFO    SenderThread:2196144 [sender.py:send_request_defer():248] send defer: 3
2021-01-16 16:09:12,643 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: defer
2021-01-16 16:09:12,643 INFO    HandlerThread:2196144 [handler.py:handle_request_defer():69] handle defer: 3
2021-01-16 16:09:12,643 DEBUG   SenderThread:2196144 [sender.py:send():117] send: summary
2021-01-16 16:09:12,644 INFO    SenderThread:2196144 [sender.py:_save_file():682] saving file wandb-summary.json with policy end
2021-01-16 16:09:12,644 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:09:12,644 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: defer
2021-01-16 16:09:12,644 INFO    SenderThread:2196144 [sender.py:send_request_defer():212] handle sender defer: 3
2021-01-16 16:09:12,644 INFO    SenderThread:2196144 [sender.py:send_request_defer():248] send defer: 4
2021-01-16 16:09:12,645 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: defer
2021-01-16 16:09:12,645 INFO    HandlerThread:2196144 [handler.py:handle_request_defer():69] handle defer: 4
2021-01-16 16:09:12,645 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:09:12,645 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: defer
2021-01-16 16:09:12,645 INFO    SenderThread:2196144 [sender.py:send_request_defer():212] handle sender defer: 4
2021-01-16 16:09:12,645 INFO    SenderThread:2196144 [dir_watcher.py:finish():279] shutting down directory watcher
2021-01-16 16:09:13,171 INFO    SenderThread:2196144 [dir_watcher.py:_on_file_modified():226] file/dir modified: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/wandb-summary.json
2021-01-16 16:09:13,171 INFO    SenderThread:2196144 [dir_watcher.py:_on_file_modified():226] file/dir modified: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/config.yaml
2021-01-16 16:09:13,172 INFO    SenderThread:2196144 [dir_watcher.py:finish():309] scan: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files
2021-01-16 16:09:13,172 INFO    SenderThread:2196144 [dir_watcher.py:finish():315] scan save: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/output.log output.log
2021-01-16 16:09:13,172 INFO    SenderThread:2196144 [dir_watcher.py:finish():315] scan save: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/requirements.txt requirements.txt
2021-01-16 16:09:13,172 INFO    SenderThread:2196144 [dir_watcher.py:finish():315] scan save: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/config.yaml config.yaml
2021-01-16 16:09:13,172 INFO    SenderThread:2196144 [dir_watcher.py:finish():315] scan save: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/wandb-metadata.json wandb-metadata.json
2021-01-16 16:09:13,172 INFO    SenderThread:2196144 [dir_watcher.py:finish():315] scan save: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/wandb-summary.json wandb-summary.json
2021-01-16 16:09:13,172 INFO    SenderThread:2196144 [sender.py:send_request_defer():248] send defer: 5
2021-01-16 16:09:13,173 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: defer
2021-01-16 16:09:13,173 INFO    HandlerThread:2196144 [handler.py:handle_request_defer():69] handle defer: 5
2021-01-16 16:09:13,184 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:09:13,184 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: defer
2021-01-16 16:09:13,184 INFO    SenderThread:2196144 [sender.py:send_request_defer():212] handle sender defer: 5
2021-01-16 16:09:13,184 INFO    SenderThread:2196144 [file_pusher.py:finish():170] shutting down file pusher
2021-01-16 16:09:13,185 INFO    SenderThread:2196144 [sender.py:send_request_defer():248] send defer: 6
2021-01-16 16:09:13,187 DEBUG   Thread-18 :2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:09:13,190 DEBUG   Thread-17 :2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:09:13,197 DEBUG   Thread-20 :2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:09:13,198 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: defer
2021-01-16 16:09:13,198 INFO    HandlerThread:2196144 [handler.py:handle_request_defer():69] handle defer: 6
2021-01-16 16:09:13,205 DEBUG   Thread-19 :2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): api.wandb.ai:443
2021-01-16 16:09:13,205 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:09:13,206 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: defer
2021-01-16 16:09:13,206 INFO    SenderThread:2196144 [sender.py:send_request_defer():212] handle sender defer: 6
2021-01-16 16:09:13,486 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:09:13,557 DEBUG   Thread-18 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:09:13,564 DEBUG   Thread-18 :2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): storage.googleapis.com:443
2021-01-16 16:09:13,568 DEBUG   Thread-17 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:09:13,574 DEBUG   Thread-17 :2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): storage.googleapis.com:443
2021-01-16 16:09:13,575 DEBUG   Thread-19 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:09:13,576 DEBUG   Thread-20 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /graphql HTTP/1.1" 200 None
2021-01-16 16:09:13,587 DEBUG   Thread-20 :2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): storage.googleapis.com:443
2021-01-16 16:09:13,587 DEBUG   Thread-19 :2196144 [connectionpool.py:_new_conn():941] Starting new HTTPS connection (1): storage.googleapis.com:443
2021-01-16 16:09:13,734 DEBUG   Thread-10 :2196144 [connectionpool.py:_make_request():442] https://api.wandb.ai:443 "POST /files/whiteashes/fedml/dn161w34/file_stream HTTP/1.1" 200 None
2021-01-16 16:09:13,735 INFO    SenderThread:2196144 [sender.py:send_request_defer():248] send defer: 7
2021-01-16 16:09:13,736 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: defer
2021-01-16 16:09:13,736 INFO    HandlerThread:2196144 [handler.py:handle_request_defer():69] handle defer: 7
2021-01-16 16:09:13,736 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:09:13,736 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: defer
2021-01-16 16:09:13,736 INFO    SenderThread:2196144 [sender.py:send_request_defer():212] handle sender defer: 7
2021-01-16 16:09:13,736 INFO    SenderThread:2196144 [sender.py:send_request_defer():248] send defer: 8
2021-01-16 16:09:13,737 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: defer
2021-01-16 16:09:13,737 INFO    HandlerThread:2196144 [handler.py:handle_request_defer():69] handle defer: 8
2021-01-16 16:09:13,737 DEBUG   SenderThread:2196144 [sender.py:send():117] send: final
2021-01-16 16:09:13,737 DEBUG   SenderThread:2196144 [sender.py:send():117] send: footer
2021-01-16 16:09:13,737 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:09:13,738 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: defer
2021-01-16 16:09:13,738 INFO    SenderThread:2196144 [sender.py:send_request_defer():212] handle sender defer: 8
2021-01-16 16:09:14,089 DEBUG   Thread-20 :2196144 [connectionpool.py:_make_request():442] https://storage.googleapis.com:443 "PUT /wandb-production.appspot.com/whiteashes/fedml/dn161w34/wandb-summary.json?Expires=1610870953&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=bXENeJuDchxxxvlmPj8HdXSi%2BqY%2FEa0Y6j1C965sFk%2F9wXOLOB1pMdmDH4Lc6sMiELq6B%2FXAbGP3OhOoU5ASI%2BaCjv6CQ1kfBKILeejdcSm6hUQMwn0yNaJmXDM2iM7Nwg6zqO1NXYD827EdYFit%2F0ZMAfYysJimTPTE0jIF0F48yNYsskXtRkzT%2BgNYzC5xG1DliiYqLgziydNYKVkm5l9h9VN6Xvi2yOZGPANesfqu4Ct2O73EDRHoxMAyhAZ3JqBXU2w7AJ8yZgoqr4X9fULccYqqRe7lU9tklZDn0TwqS%2FGXF1pzH98DmD3nToWgQjcXYubnVG1vTFGvzP4DPQ%3D%3D HTTP/1.1" 200 0
2021-01-16 16:09:14,091 INFO    Thread-20 :2196144 [upload_job.py:push():131] Uploaded file /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/wandb-summary.json
2021-01-16 16:09:14,091 DEBUG   Thread-18 :2196144 [connectionpool.py:_make_request():442] https://storage.googleapis.com:443 "PUT /wandb-production.appspot.com/whiteashes/fedml/dn161w34/requirements.txt?Expires=1610870953&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=Sw8eosHNCciIh8FhwTxPCnB%2FRuwat7BY4nXSyftPTn9Typtg2dU3LXY5LCO9iH%2BIIc%2BWlCndI8S8FR%2Ff7E6MevcE%2BYCQ4o1dhaKzX3N78whvdoVBd4p7umDcr%2FoVahOIqHh270W2cRx3P1dobpqhmbX6tXCYxQgD6b7Ot9RIX8hef29y1lj8MuVDkBoonY1L4DvoPZSJnMemo6E4MfQFkDIIcKZzdyr8eKsJcoEw49IwVXQVm50cFlR9vpmJ9u9PAN%2Fq0FI%2BMKydUM4r2914ua8nTqghrOTqdEDsjF3qcTllHNbJnUkAeJwSpwcjaCXmd7I0qS3NgNxXxXQ4ADP7nA%3D%3D HTTP/1.1" 200 0
2021-01-16 16:09:14,092 INFO    Thread-18 :2196144 [upload_job.py:push():131] Uploaded file /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/requirements.txt
2021-01-16 16:09:14,093 DEBUG   Thread-19 :2196144 [connectionpool.py:_make_request():442] https://storage.googleapis.com:443 "PUT /wandb-production.appspot.com/whiteashes/fedml/dn161w34/config.yaml?Expires=1610870953&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=D7hzIrXgBAhCzx6dk44YBgmN22mo4MTQg2G%2BEkXmDHJ4OAa5cY%2FA4vMWiIWPr68VrZNLR6RR8uqDkgPq161xXQiZ%2BFOTynrhbehmEyNwkW8qYXDTQndtkpm%2ByAye4BATmBPtmcIka9fGN64%2F6AUne%2FJGIUfo2qW%2F%2Fc1I2fVnoWmjoGHFvfwwqEBn8oCGQJxSKltuxKGH9AJZ%2BVJ6I1WMMHAroSt6iMGjq3Y350Vb37pILQHhRsDUou2BFzmWXyoU%2BIEqwlKTqUo7D9qcJESTlvjduqxtUsl54dLhlqhNVEcbaVNJHIObf%2FxogWB6zv7eXAlVgNrvOKDV0BmDrmPyGA%3D%3D HTTP/1.1" 200 0
2021-01-16 16:09:14,094 INFO    Thread-19 :2196144 [upload_job.py:push():131] Uploaded file /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/config.yaml
2021-01-16 16:09:14,096 DEBUG   Thread-17 :2196144 [connectionpool.py:_make_request():442] https://storage.googleapis.com:443 "PUT /wandb-production.appspot.com/whiteashes/fedml/dn161w34/output.log?Expires=1610870953&GoogleAccessId=wandb-production%40appspot.gserviceaccount.com&Signature=CGMqsDAMETaU%2FWaA2RQ6j3ua7Z8IWfj%2BorSggd5rK9bG4U2bLaKygDIirkfEkZo6BA4SpWq6n2ljUKhBaEyVS9FKffbnjnqgnyTHqeYF5bnX5eksq8C8w9lJC%2FtqEsV9OGP8LdfxipieFNPibcqyyrPAqGEO2mpLYWkLLBlj0ZCwelbCso1c0pBtPtcJWheSwvK3S51OHlx3y%2BJxS4HJA9mFFftrEt2%2FLrmXom0A9dt%2B3CnDOtva50uE7LDV2jztOU%2FuDU8HPZaQezxBWrNA7SGzK%2BcJfyQaqIx%2BO7sFp1Tujroam9Sx9F2gmiReLgyyo8wcQXt5vjBRR9isTcguMA%3D%3D HTTP/1.1" 200 0
2021-01-16 16:09:14,097 INFO    Thread-17 :2196144 [upload_job.py:push():131] Uploaded file /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/files/output.log
2021-01-16 16:09:14,561 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: poll_exit
2021-01-16 16:09:14,561 DEBUG   SenderThread:2196144 [sender.py:send():117] send: request
2021-01-16 16:09:14,561 DEBUG   SenderThread:2196144 [sender.py:send_request():126] send_request: poll_exit
2021-01-16 16:09:14,561 INFO    SenderThread:2196144 [file_pusher.py:join():175] waiting for file pusher
2021-01-16 16:09:14,562 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: get_summary
2021-01-16 16:09:14,563 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: sampled_history
2021-01-16 16:09:14,563 DEBUG   HandlerThread:2196144 [handler.py:handle_request():55] handle_request: shutdown
2021-01-16 16:09:14,564 INFO    HandlerThread:2196144 [handler.py:finish():274] shutting down handler
2021-01-16 16:09:14,737 INFO    WriterThread:2196144 [datastore.py:close():258] close: /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_160503-dn161w34/run-dn161w34.wandb
2021-01-16 16:09:15,562 INFO    SenderThread:2196144 [sender.py:finish():756] shutting down sender
2021-01-16 16:09:15,562 INFO    SenderThread:2196144 [file_pusher.py:finish():170] shutting down file pusher
2021-01-16 16:09:15,562 INFO    SenderThread:2196144 [file_pusher.py:join():175] waiting for file pusher
2021-01-16 16:09:15,771 INFO    MainThread:2196144 [internal.py:handle_exit():139] Internal process exited
